seshdash/static/sesh-ui/css/sesh.css

```diff
@@ -241,3 +241,6 @@ div.awesomplete > ul {
   border:2px solid #CA2F2F;
   box-shadow: 10px red;
 }
+.page-header{
+  font-weight: 200;
```

Comment:
Frank do you also need to change the font?
Is this the same font as in the historical page?